plugins/sinks/kafka/sink.go

```diff
@@ -2,79 +2,67 @@ package kafka
 
 import (
 	"context"
-	"encoding/json"
-	"errors"
-	"fmt"
-
 	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"github.com/golang/protobuf/proto"
 	"github.com/odpf/meteor/core"
 	"github.com/odpf/meteor/core/sink"
+	"github.com/pkg/errors"
 )
 
 type Sink struct{}
 
-func init() {
-	if err := sink.Catalog.Register("kafka", New()); err != nil {
-		panic(err)
-	}
-}
-
 func New() core.Syncer {
 	return new(Sink)
 }
 
-// Kafka sink
 func (s *Sink) Sink(ctx context.Context, config map[string]interface{}, out <-chan interface{}) (err error) {
+	if config["brokers"] == nil {
```

Comment:
for handling config, you can use this [utility](https://github.com/odpf/meteor/blob/main/utils/config.go) to help you build, validate and use default values.

sample usage can be found here:
https://github.com/odpf/meteor/blob/main/plugins/extractors/mysql/extractor.go#L38

as for now the error message is not being output clearly which Im currently working on right now.

Reply:
Got it, using the util!